packages/iTwinUI-react/src/core/index.ts

```diff
@@ -5,6 +5,9 @@
 export { Alert } from './Alert';
 export type { AlertProps } from './Alert';
 
+export { Backdrop } from './Backdrop';
+export type { BackdropProps } from './Backdrop';
+
```

Comment:
so we don't want to expose `<Dialog>` component just yet?

Reply:
Maybe the main reason if we see something while implementing draggable dialog.